code/game/sound.dm

```diff
@@ -177,7 +177,7 @@
 			if("goose")
 				soundin = pick('sound/creatures/goose1.ogg', 'sound/creatures/goose2.ogg', 'sound/creatures/goose3.ogg', 'sound/creatures/goose4.ogg')
 			if("warpspeed")
-				soundin = 'sound/effects/hyperspace_begin.ogg'
+				soundin = 'sound/runtime/hyperspace/hyperspace_begin.ogg'
```

Comment:
Should this be in runtime if it's just included.

Reply:
It's accessed using double quotes by shuttles.